src/Appium.Net/Appium/Service/AppiumCommandExecutor.cs

```diff
@@ -53,14 +54,16 @@ internal AppiumCommandExecutor(AppiumLocalService service, TimeSpan timeForTheSe
             ClientConfig = clientConfig;
         }
 
-        public Response Execute(Command commandToExecute)
+        public Response Execute(Command commandToExecute) => Task.Run(() => ExecuteAsync(commandToExecute)).GetAwaiter().GetResult();
```

Comment:
Consider using a block of code.
 ```suggestion
        public Response Execute(Command commandToExecute)
        {
            return Task.Run(() => ExecuteAsync(commandToExecute)).GetAwaiter().GetResult();
        }
```

Reply:
@kelmelzer let me know if you approve the suggestion. Not critical

Reply:
All good. I just like that operator but your suggestion makes sense in the overall scheme of the code :)